toolchain/diagnostics/diagnostic.h

```diff
@@ -105,6 +109,10 @@ struct Message {
 
   // Returns the formatted string. By default, this uses llvm::formatv.
   std::function<auto(const Message&)->std::string> format_fn;
+
+  // Storage for a custom snippet, if one is being used.
+  // TODO: Find a better way to store this.
+  std::shared_ptr<std::string> snippet_storage = nullptr;
```

Comment:
Doesn't seem like this ended up getting used -- looks like we'1re just storing the snippet inside the `Loc` above, which seems reasonable?

Reply:
Thanks, removed.